site/_posts/2021-02-04-single-sign-on.md

```diff
@@ -0,0 +1,655 @@
+---
+layout: blog-post
+title: Setting up single sign-on with FusionAuth
```

Comment:
Setting up single sign-on (SSO) with FusionAuth

Not a huge thing, but we'll be able to hit both keywords if you include them in the title and URL

Reply:
Sounds good!